series1_float.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+int main()
+
+{ 
+   float n,init_s=2, sum = 0;
+    cout << "Please enter the value for n (max ) : ";
```

Comment:
using io is not a good idea for this homework, since it is 1. not asked and 2. makes the python part very difficult